src/providers/definition/common.ts

```diff
@@ -48,7 +48,7 @@ export const viewSuggestions: DefinitionSuggestion[] = [
 	{ // id
 		regExp: /id=["'][\s0-9a-zA-Z-_^]*$/,
 		files (): string[] {
-			return getRelatedFiles('tss');
+			return getRelatedFiles('tss', false);
```

Comment:
I think this will remove app.ts for the definition providers too. We probably want to keep that.

Maybe the filtering should be done where `files` is called?